Patch by Manu Abraham and Gerd Knorr:

Remove redundant bttv_reset_audio() which caused the computer to
freeze with some bt8xx based DVB cards when loading the bttv driver.

Signed-off-by: Johannes Stezenbach <js@linuxtv.org>

 drivers/media/video/bttv-cards.c |    2 --
 1 files changed, 2 deletions(-)

--- a/drivers/media/video/bttv-cards.c	2005-03-26 04:28:20.000000000 +0100
+++ b/drivers/media/video/bttv-cards.c	2005-04-07 13:03:27.000000000 +0200
@@ -2718,8 +2718,6 @@
         }
 	btv->pll.pll_current = -1;
 
-	bttv_reset_audio(btv);
-
 	/* tuner configuration (from card list / autodetect / insmod option) */
  	if (UNSET != bttv_tvcards[btv->c.type].tuner_type)
 		if(UNSET == btv->tuner_type)
